### PR TITLE
feat: Add ferris wheel project

### PR DESCRIPTION
**Pull request by moshoodabdullahi**
# This PR adds a ferris wheel webpage using animate elements, transform and speed.

## Built With

- HTML5 elements
  - Block-level elements such as `div`.
- CSS properties
  - CSS Position `absolute`,
  - CSS transform such as `rotate`,
  - CSS Variables for reusable CSS values,
  - CSS speed adjust such as `infinite`, and so on,
  - CSS animation such as `animation delay`, `timing function` and so on,
  - CSS Background for page aesthetic using `background-color`,
  - CSS key frames for specifying animation code using `@keyframes` rule.
- Linters
  - [Prettier](https://prettier.io/),
  - [Stylelint](https://stylelint.io/).
- GitHub Action

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,36 @@
...
-- frameworks,
-- technologies used
+- HTML5 elements
+  - Block-level elements such as `div`.
+- CSS properties
+  - CSS Positioning `absolute`,
+  - CSS transform such as `rotate`,
+  - CSS Variables for reusable CSS values,
+  - CSS speed adjust such as `infinite`, `10s`, and so on,
+  - CSS animate such as `cabins`, `ease-in-out`, and so on,
```
Comment (Yinkusadel) [inline README.md]: I suggest you make use of the CSS properties for animations here rather than the values

File: README.md [hunk 2]
```diff
@@ -1,27 +1,36 @@
...
 ## Built With
 
-- Major languages,
-- frameworks,
-- technologies used
+- HTML5 elements
+  - Block-level elements such as `div`.
+- CSS properties
+  - CSS Positioning `absolute`,
+  - CSS transform such as `rotate`,
```
Comment (Yinkusadel) [inline README.md:13]: And this should basically fall under typography

**Review by Yinkusadel — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (moshoodabdullahi) [reply README.md:13]: Do you mean text-transform there but I use transform literally it is not typography

File: README.md (hunk 1 above)
Comment (moshoodabdullahi) [reply README.md]: Done with that now.

**Pull request merged**